cmd/system-probe/modules/eventmonitor_linux.go

```diff
@@ -28,10 +29,27 @@ var EventMonitor = module.Factory{
 	},
 }
 
+const (
+	eventMonitorID          = "PROCESS_MONITOR"
+	eventMonitorChannelSize = 500
+)
+
+var (
+	eventTypes = []consumers.ProcessConsumerEventTypes{
+		consumers.ExecEventType,
+		consumers.ExitEventType,
+	}
+)
+
 func createProcessMonitorConsumer(evm *eventmonitor.EventMonitor, config *netconfig.Config) (eventmonitor.EventConsumer, error) {
 	if !usmconfig.IsUSMSupportedAndEnabled(config) || !usmconfig.NeedProcessMonitor(config) || usmstate.Get() != usmstate.Running {
 		return nil, nil
 	}
 
-	return procmon.NewProcessMonitorEventConsumer(evm)
+	consumer, err := consumers.NewProcessConsumer(eventMonitorID, eventMonitorChannelSize, eventTypes, evm)
```

Comment:
The caller of `createProcessMonitorConsumer` calls `RegisterEventConsumer` but this `NewProcessConsumer` also calls `RegisterEventConsumer` so the consumer is being registered twice.